### DISCUSSION TITLE
Insights Middleware

### DISCUSSION THREAD

**Discussion by MoeMamdouh** [algolia/instantsearch#5356] [Q&A] [answered]
Can I use Insights Middleware with react-native front end?

**Comment by Haroenv** [comment]
This is tracked in https://github.com/algolia/instantsearch.js/issues/5148, but hadn't been asked before. I'm currently making a pull request to allow it to work.

**Comment by dhayab** [comment, marked as answer]
Hi, you can use `createInsightsMiddleware()` from instantsearch.js@4.49.1 to add insights support in your react-native front end.
